src/main/catalog-sources/__test__/kubeconfig-sync.test.ts

```diff
@@ -54,10 +55,9 @@ describe("kubeconfig-sync.source tests", () => {
     di.override(kubectlDownloadingNormalizedArchInjectable, () => "amd64");
     di.override(normalizedPlatformInjectable, () => "darwin");
 
-    di.override(clusterStoreInjectable, () =>
-      ClusterStore.createInstance({ createCluster: () => null as never }),
-    );
-
+    di.permitSideEffects(fsInjectable);
```

Comment:
Doesn't seem right to introduce real `fs` to unit tests.

Reply:
I am not, this test previously (and still does) use `mockFs`. I just changed how it is used.